public/stylesheet.css

```diff
@@ -954,11 +954,11 @@ body:not(.dark):not(.light) #color-scheme-system:not(:disabled) {
   position: relative;
   color: var(--color-text-default);
   top: -58px;
-  left: -80px;
+  left: -23px;
```

Comment:
Reverted these changes because are turning the extensions button unusable.

The language switcher must be refactored to support more languages 🤔